k8s-application/envs.tf

```diff
@@ -0,0 +1,29 @@
+locals {
+  app_envs = lookup(local.app_block, "envs", {})
+
+  // extract flattened list of all environmetn variables, while mutating to append dependency (connection) name to front of path
+  merged_envs_connections = flatten([for env_key, env_path in local.app_envs : [
```

Comment:
merged_envs_params and merged_envs_connections are doing exactly the same thing. why can't we do that in one pass?

Reply:
I split them because of the `jq_query` below. One takes `connections` and the other `params`. Also, if a param has the same name as a connection, they would conflict. 

```
params:
  properties:
    database:
      type: string
      
connections:
  properties:
    database:
      $ref: massdriver/database
      
app:
  envs:
    DATABASE_NAME: params.database
    DATABASE_HOST: connections.database.data.authentication.hostname            
```

Reply:
couldn't we merge both lists as a map and only use one jq query data resource?

Reply:
If a param has the same name as a connection, terraform implodes.

Reply:
Ahh fair enough. They'd have the same json key just different files but both vars in tf land. I'll update and sync with Cory